app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,18 +11,153 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-searchbar {
```

Comment:
I think this search specific code would be better in a specific file. something like shop_search.css

Reply:
Done.